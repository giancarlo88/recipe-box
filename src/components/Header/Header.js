import React from 'react'
import styled from 'styled-components'


const Header = styled.header`
background-color: rgba(125, 30, 50, .5);
color: green;
`

export default () => (
  <Header>
    <h1 className="App-title">Welcome to React</h1>
  </Header>
)
