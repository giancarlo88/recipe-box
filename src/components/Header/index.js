import React from 'react'
import styled from 'styled-components'


const StyledHeader = styled.header`
align-items: center;
background-color: rgba(125, 30, 50, .5);
display: flex;
height: 75px;
justify-content: center;
text-align: center;
`

const HeaderTitle = styled.h1`
font-family: 'Montserrat';
font-size: 36px;
`

const Header = () => (
  <StyledHeader>
    <HeaderTitle>Recipes</HeaderTitle>
  </StyledHeader>
)

export default Header

