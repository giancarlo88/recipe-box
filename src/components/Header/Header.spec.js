import React from 'react'
import Header from './Header'
import { create } from 'react-test-renderer'


describe('Header', () => {

  it('should match a snapshot', () => {
    const tree = create(<Header/>)
    expect(tree).toMatchSnapshot()
  });

});
