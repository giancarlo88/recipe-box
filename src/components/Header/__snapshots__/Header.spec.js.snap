// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match a snapshot 1`] = `
<header
  className="sc-bdVaJa ePfJUz"
>
  <h1>
    The Recipe Box
  </h1>
</header>
`;
